src/scripts/main.js

```diff
@@ -1 +1,18 @@
 'use strict';
+
+const largeImage = document.getElementById('largeImg');
+const gallery = document.querySelector('.gallery');
+
+gallery.addEventListener('click', () => {
+  event.preventDefault();
+
+  const target = event.target;
```

Comment:
The `event` object is not defined in the current scope because it was not passed to the event listener function. Make sure to pass `event` as a parameter to the function.